common/persistence/cassandra/matching_task_store.go

```diff
@@ -39,6 +39,99 @@ import (
 	"go.temporal.io/server/common/primitives/timestamp"
 )
 
+const (
+	templateCreateTaskQuery = `INSERT INTO tasks (` +
+		`namespace_id, task_queue_name, task_queue_type, type, task_id, task, task_encoding) ` +
+		`VALUES(?, ?, ?, ?, ?, ?, ?)`
+
+	templateCreateTaskWithTTLQuery = `INSERT INTO tasks (` +
+		`namespace_id, task_queue_name, task_queue_type, type, task_id, task, task_encoding) ` +
+		`VALUES(?, ?, ?, ?, ?, ?, ?) USING TTL ?`
+
+	templateGetTasksQuery = `SELECT task_id, task, task_encoding ` +
+		`FROM tasks ` +
+		`WHERE namespace_id = ? ` +
+		`and task_queue_name = ? ` +
+		`and task_queue_type = ? ` +
+		`and type = ? ` +
+		`and task_id > ? ` +
+		`and task_id <= ?`
```

Comment:
should this be [Inclusive, exclusive) as well?

Reply:
Yeah, I added a TODO for this on TaskManager's interface definition. But I don't want to touch matching tasks in this PR. 
Changes here is simply move matching tasks related queries to this file.